website/docs/docs/build/incremental-models.md

```diff
@@ -310,5 +310,61 @@ select ...
 
 </File>
 
+<VersionBlock firstVersiont="1.4">
+
+### About incremental_predicates
+
+`incremental_predicates` is an advanced use of incremental models, where data volume is large enough to justify additional investments in performance. This config accepts any valid SQL expression, however dbt does not check the syntax. 
+
+For example, this is a pattern we might expect to see on Snowflake:
+
+```yml
+models:
+  - name: my_incremental_model
+    config:
+      materialized: incremental
+      unique_key: id
+      cluster_by: ['session_start']  # this will affect how the data is stored on disk, and indexed to limit scans
+      incremental_strategy: merge
+      # limit scan of the existing table to the last 7 days of data
+      incremental_predicates: "DBT_INTERNAL_DEST.session_start > datediff(day, -7, current_date)"
+```
+
+This will template a `merge` statement like:
```

Comment:
where will this template a merge statement? not sure if i understand where the user will see the code below?

Reply:
Good question, let me check with the SME's on this

Reply:
This would be visible in the `dbt.log` file only (and their warehouse query history) -- this is currently true of incremental models generally. the compiled code in the `target` directory will contain the SQL that creates the `temp_table_with_new_records`